Source/Fuse.Maps/MapTile.uno

```diff
@@ -0,0 +1,285 @@
+using Fuse.Elements;
+using Uno.UX;
+using Uno;
+using Fuse.Maps;
+using Uno.Collections;
+
+namespace Fuse.Controls
+{
+	/** A polyfill for Native @MapView
+
+	This is not meant to be a full implementation, but just something that will help you when developing on local preview.
+
+	This should also work as a standalone component, so it can be used on native.block
+
+	This uses the OSM tile server, and that has both a usage policy https://operations.osmfoundation.org/policies/tiles/ and a license https://www.openstreetmap.org/copyright .
+
+	The mappin is from wikipedia https://commons.wikimedia.org/wiki/File:Map_pin_icon.svg and is Creative Commons Attribution-Share Alike 3.0 Unported. 
```

Comment:
That was my thoughts. The reason is:

- This is not production ready yet. So it will only be used in development.
- If you release a high traffic app, you would need either your own mapserver (something like https://github.com/ncareol/osm-tiles-docker ), or a service (like http://www.geofabrik.de/ ), both of which would change the attribution requirement.